<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemBrowser</class>
 <widget class="QDialog" name="ItemBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1000">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QPushButton" name="addItemBut">
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteItemBut">
       <property name="text">
        <string>Delete Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="turnSheetBut">
       <property name="toolTip">
        <string>Parse item data from a turn sheet</string>
       </property>
       <property name="text">
        <string>Import Turn Sheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importBut">
       <property name="toolTip">
        <string>Import items from an exported xml file</string>
       </property>
       <property name="text">
        <string>Import XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBut">
       <property name="toolTip">
        <string>Export the items database to a separate file</string>
       </property>
       <property name="text">
        <string>Export XML</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QPlainTextEdit" name="descEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Category:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="categoryCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="subcatLabel">
        <property name="text">
         <string>Sub-Category:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="subcatCombo"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="tonsLabel">
        <property name="text">
         <string>Tons:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="tonsSpin"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Structure:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="structureSpin"/>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QPushButton" name="saveItemBut">
        <property name="text">
         <string>Save Item</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="comps">
         <attribute name="title">
          <string>Components</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
            <item>
             <widget class="QPushButton" name="addMatBut">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteMatBut">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="MaterialsWidget" name="materialsTable">
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DropOnly</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="effects">
         <attribute name="title">
          <string>Effects</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
            <item>
             <widget class="QPushButton" name="addEffectBut">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteEffectBut">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="effectsTable"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialsWidget</class>
   <extends>QTableWidget</extends>
   <header>MaterialsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>nameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descEdit</sender>
   <signal>textChanged()</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>categoryCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subcatCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tonsSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>structureSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>materialsTable</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>effectsTable</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>ItemBrowser</receiver>
   <slot>fieldsChangedSot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fieldsChangedSot()</slot>
 </slots>
</ui>
